### PR TITLE
API-7729 Add list_webhook_names method

### PR DESCRIPTION
**Pull request by kogierman**
## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/API-7729-list-webhook-names)
- [Jira](https://livechatinc.atlassian.net/browse/API-7729)

## 📓 Description

Docs for new method `list_webhook_names`

### Content

- New content

### CODE REVIEW

File: content/management/configuration-api/v3.3/index.mdx [hunk 1]
```diff
@@ -3634,6 +3634,46 @@ curl -X POST \
...
+      }'
+```
+
+</CodeSample>
+
+<CodeResponse type="configuration" version="v3.3" json="listWebhookNames"/>
+
+</Code>
+
+</Section>
```
Comment (ale-kacperczyk) [inline content/management/configuration-api/v3.3/index.mdx:3675]: The build has failed. I can't see the <Section> opening tag.

File: content/management/configuration-api/v3.3/index.mdx [hunk 2]
```diff
@@ -3634,6 +3634,52 @@ curl -X POST \
 
 </Section>
 
+<Section>
+
+<Text>
+
+### List Webhook Names
+
+Lists all webhooks that are supported in given API version. This method requires no authorization.
```
Comment (ale-kacperczyk) [inline content/management/configuration-api/v3.3/index.mdx:3643]:
```suggestion
Lists all webhooks that are supported in q given API version. This method requires no authorization.
```

File: content/management/configuration-api/v3.3/index.mdx [hunk 3]
```diff
@@ -3634,6 +3634,52 @@ curl -X POST \
...
+|                 |                                                                            |
+| --------------- | -------------------------------------------------------------------------- |
+| Method URL      | `https://api.livechatinc.com/v3.3/configuration/action/list_webhook_names` |
+| Required scopes | -                                                                          |
+
+#### Request
+
+| Parameter      | Required | Data type | Notes                                  |
+| -------------- | -------- | --------- | -------------------------------------- |
+| `version`      | No       | `string`  | Defaults to current stable API version |
```
Comment (ale-kacperczyk) [inline content/management/configuration-api/v3.3/index.mdx:3656]:
```suggestion
| `version`      | No       | `string`  | Defaults to the current stable API version |
```

File: content/management/configuration-api/v3.3/index.mdx [hunk 4]
```diff
@@ -3634,6 +3634,52 @@ curl -X POST \
...
+
+#### Response
```
Comment (ale-kacperczyk) [inline content/management/configuration-api/v3.3/index.mdx:3658]:
```suggestion
```

File: content/management/configuration-api/v3.3/index.mdx (hunk 2 above)
Comment (kogierman) [reply content/management/configuration-api/v3.3/index.mdx:3643]: q? :)

Comment (ale-kacperczyk) [reply content/management/configuration-api/v3.3/index.mdx:3643]: sorry, corrected.

**Review by ale-kacperczyk — APPROVED** [review]

**Review by bkubiak — APPROVED** [review]

**Pull request merged**